<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="IconProcessing">
     <property name="text">
      <string>Показывать иконки на Кнопках (Эсперементально)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16666666</width>
       <height>16666666</height>
      </size>
     </property>
     <property name="title">
      <string>Маска</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="Mask00"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Mask01"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="Mask02"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="Mask03"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="Mask04"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Mask10"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Mask11"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="Mask12"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="Mask13"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="Mask14"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="Mask20"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Mask21"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="Mask22"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="Mask23"/>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="Mask24"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="Mask30"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="Mask31"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="Mask32"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="Mask33"/>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="Mask34"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="Mask40"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="Mask41"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="Mask42"/>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="Mask43"/>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="Mask44"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSlider" name="LaplasSLider">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="SliderValue">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="HalfGamma">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSlider" name="GammaSLider">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="GammaValue">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>0.3</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
